src/contents/loginWarning.svelte

```diff
@@ -0,0 +1,110 @@
+<script context="module" lang="ts">
+  import cssText from "data-text:~/contents/loginWarningOverlay.css"
+  import type { PlasmoCSConfig } from "plasmo"
+
+  export const config: PlasmoCSConfig = {
+    matches: ["<all_urls>"],
+    exclude_matches: [
+      "*://*.steamcommunity.com/*",
+      "*://*.steampowered.com/*",
+      "*://*.steam.tv/*"
```

Comment:
What is steam.tv?

Reply:
Some kind of website that loads the same content, so it triggered the warning.
Inspiration from https://github.com/Jessecar96/Steam-Community-Suite/blob/8ddde18105d85cb1e6d83a0ee7414cec2ccf7d49/src/script.js#L54